src/containers/DataPackageViewContainer.js

```diff
@@ -33,8 +27,8 @@ export class DataPackageViewContainer extends React.Component {
 
   convertData(data, dp) {
```

Comment:
Can you please add some comments, what this method does. And for other methods, as possible please.